1 Standard/002_Ivysaur.yaml

```diff
@@ -35,3 +35,22 @@ ball: [Great]
 nature: Naive
 ivs: 31
 evs: {hp: 92, atk: 56, def: 100, spA: 200, spD: 32, spe: 28}
+---
+species: Ivysaur
+setname: Project M
+tags: [standard]
+item: [Null]
+ability: [Overgrow]
+moves:
+    - [Synthesis]
+    - [Razor Leaf]
+    - [Vine Whip]
+    - [Seed Bomb]
```

Comment:
More move order problems, mono grass attacking isn't ideal.

Reply:
This is based off the Project M Ivysaur, like the Smash advanced movesets. I’ll see what I can do.